### PR TITLE
[Back End] npm installed axios and updated PUT /api/courses endpoints

### PR DESCRIPTION
**Pull request by teresal92**
Screen Shot 2022-03-22 at 6 06 25 PM

Major Updates:
1. Updated PUT request route for /pages/api/courses/[course.id].js to split up updateCourseInfo and updateCourseMenteeList to be separate function calls. The PUT requests requires different queries so this is a solution to help clean up the routes in the back-end.
- PUT request to updateCourseInfo (course name, start_time and end_time) will go to route /pages/api/courses/[course_id].js
- PUT requests to updateCourseMenteeList will go to /pages/api/courses/mentees/[course_id].js

2. NPM installed axios and imported to apiCalls.js in utils/api folder

Pending:
- Writing out all the apiCalls to be exported and used in the front end

### CODE REVIEW

File: pages/api/courses/mentees/[course_id].js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      //     course_id,
+      //     course_name,
+      //     mentor_id,
+      //     mentor_name,
+      //     mentee_id,
+      //     mentee_name,
+      //     mentee_status: 'pending' // pending initially, accepted, declined
+      //   }
+      // }
+
```
Comment (tiffanyyv) [inline pages/api/courses/mentees/[course_id].js:56]: Not sure if you meant to delete this or if we're planning to use it later!

**Review by tiffanyyv — COMMENTED** [review]
Looks good!

File: pages/api/courses/mentees/[course_id].js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        }
+      })
+    });
+    console.log('querySnapshot: ', querySnapshot)
+    res.status(200).json(`Successfully added student to course ${course_id}`);
+  } catch (err) {
+    res.status(400).send(`Error adding student to course ${course_id}: ${err}`);
+  }
+}
+
```
Comment (samanthavpham) [inline pages/api/courses/mentees/[course_id].js:34]: can we also add to the commented out code that this case on hold until after the demo?

File: pages/api/users/[user_id].js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 // GET: /pages/api/users/[user_id].js
-
 import { db } from '../../../utils/api/firebase.config';
 import { collection, doc, getDoc } from "firebase/firestore";
 
 export default async function getUserInfo(req, res) {
   const { user_id } = req.query;
-  console.log('uid: ', user_id);
+  console.log('req: ', req);
```
Comment (samanthavpham) [inline pages/api/users/[user_id].js:7]: can we remove this console.log

**Review by samanthavpham — CHANGES_REQUESTED** [review]
great job condensing the routes to query from the users table - just some minor notes to remove some console.logs and include an explanation for large blocks of commented out code. :)

**Review by samanthavpham — APPROVED** [review]
These notes can be applied in v2 if easier

**Review by cmjosephs — APPROVED** [review]
😃

**Pull request merged**